src/modules/launcher/Plugins/Microsoft.Plugin.Calculator.UnitTest/ExtendedCalculatorParserTests.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using Microsoft.Plugin.Calculator;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using NUnit.Framework;
+using Wox.Plugin;
+
+namespace Microsoft.Plugin.Calculator.UnitTests
+{
+    [TestFixture]
+    public class ExtendedCalculatorParserTests
+    {
+        [TestCase("42")]
+        [TestCase("test")]
+        public void SingleNumberNoResult(string input)
```

Comment:
Can you please use the test naming convention that we generally use. You can find examples [here ](https://github.com/microsoft/PowerToys/blob/v0.21.1/src/modules/launcher/Wox.Test/ResultsViewModelTest.cs#L15). It is generally of the form `<function being tested>_<expected results>_<under conditions>`

Reply:
I hope the naming convention is better now.

Reply:
Yes it is! Thanks for these changes 💯